src/api/test-controller.js

```diff
@@ -230,6 +232,27 @@ export default class TestController {
 
         return selector();
     }
+
+    _setNativeDialogHandler$ (fn, dependencies) {
```

Comment:
It's better pass `options` for consistency here. It should also accept client function as `fn`